gitlab/objects.py

```diff
@@ -1249,6 +1249,18 @@ class ProjectKey(GitlabObject):
     requiredUrlAttrs = ['project_id']
     requiredCreateAttrs = ['title', 'key']
 
+    def enable(self, key_id, project_id):
+        """Enable a deploy key for a project."""
+        url = '/projects/%s/deploy_keys/%s/enable' % (project_id, key_id)
+        r = self.gitlab._raw_post(url)
+        raise_error_from_response(r, GitlabProjectDeployKeyError, 201)
+
+    def disable(self, key_id, project_id):
+        """Disable a deploy key for a project."""
+        url = '/projects/%s/deploy_keys/%s/disable' % (project_id, key_id)
+        r = self.gitlab._raw_delete(url)
+        raise_error_from_response(r, GitlabProjectDeployKeyError, 201)
```

Comment:
No, the expected code is 201. Check:
https://docs.gitlab.com/ce/api/deploy_keys.html

```
Enable a deploy key

Enables a deploy key for a project so this can be used. Returns the enabled key, with a status code 201 when successful.

curl --request POST --header "PRIVATE-TOKEN: 9koXpg98eAheJpvBs5tK" https://gitlab.example.com/api/v3/projects/5/deploy_keys/13/enable
Attribute	Type	Required	Description
id	integer	yes	The ID of the project
key_id	integer	yes	The ID of the deploy key
Example response:

{
   "key" : "ssh-rsa AAAA...",
   "id" : 12,
   "title" : "My deploy key",
   "created_at" : "2015-08-29T12:44:31.550Z"
}
```